specutils/io/asdf/schemas/astropy.org/specutils/spectra/spectrum1d-1.0.0.yaml

```diff
@@ -2,7 +2,6 @@
 ---
 $schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
 id: "http://astropy.org/schemas/specutils/spectra/spectrum1d-1.0.0"
-tag: "tag:astropy.org:specutils/spectra/spectrum1d-1.0.0"
```

Comment:
Move this whole schema file into just `schemas` rather than the unnecessary set of folders.